virtcontainers/fc.go

```diff
@@ -307,12 +316,21 @@ func (fc *firecracker) vmRunning() bool {
 	}
 }
 
-func (fc *firecracker) checkVersion(vmmInfo *fcmodels.InstanceInfo) error {
-	if vmmInfo == nil || vmmInfo.VmmVersion == nil {
-		return fmt.Errorf("Unknown firecracker version")
+func (fc *firecracker) checkVersion() error {
+	args := []string{"--version"}
```

Comment:
this function is called several times in `waitVMMRunning`, may be we should get the version once (at `init()` ? ) instead of executing `firecracker` several times, what do you think?

Reply:
I'm not sure that we could get version number in `init()`, which is called during the first importing package time. The path of  `firecracker` binary may not be tracked back to that early time.
But you're right, we should store it somewhere for later frequent use. ;).